src/components/dialog/components.test-pw.tsx

```diff
@@ -99,3 +100,64 @@ export const DialogWithOpenToastsBackgroundScrollTest = () => {
     </Box>
   );
 };
+
+export const DialogWithStepSequence = (props: Partial<DialogProps>) => {
```

Comment:
**comment** (non-blocking): I'm not sure why we've introduced a more complex story, and highlighted the `StepSequence` part in its name, when the purpose seems to be to use it to test the new `greyBackground` prop. If that's the intention I'd make a very simple dialog example, with the `greyBackground` prop passed explicitly, and call it something like `DialogWithGreyBackground`.